### PR TITLE
Update vudu13's slide caption

### PR DESCRIPTION
**Pull request by vudu13**
This PR updates the caption for @vudu13's slide.

- [x] Vudu13's slide
  - [x] item1
  - [x] item2
- [x] Stuff and things
  - [x] stuff
  - [x] things

#2

### CODE REVIEW

**Comment by vudu13** [comment]
🦑

File: _slides/04-vudu13.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+WHOA! 💫
```
Comment (allthedoll) [inline _slides/04-vudu13.md:6]:
```suggestion
WHOA! 💫 💫
```

File: _slides/04-vudu13.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![luchadortocat](https://octodex.github.com/images/luchadortocat.png)
```
Comment (allthedoll) [inline _slides/04-vudu13.md:8]: 😱

**Review by allthedoll — APPROVED** [review]
LGTM! 💪🏼

**Pull request merged**